### DISCUSSION TITLE
I want to learn about time steps and load steps

### DISCUSSION THREAD

**Discussion by shihaoye** [Anura3D/Anura3D_OpenSource#198] [Q&A]
Dear all,
How should I set it up for this situation. Or is there a tutorial on how to set time steps and load steps.
![7049a9d2b5fabe2e9f2dd349e2ebefa](https://github.com/Anura3D/Anura3D_OpenSource/assets/162889589/812339a9-ec92-47d8-a404-71b9f30bfd32)

Best regards,
Jack

**Comment by EmreDumlu-UM** [comment]
Only Anura3D arranges the timesteps for each loadstep. But you can increase and decrease by changing the courant number.  You may like to look at .ENG file to get further information.
In this case for the error, Anura3D checks the surrounding elements in terms of RecursiveDepth and max number of recursions (NRECURSIONS=50). For example, your particles seem to approach a fixity (i.e corner node), and never complete the recursion (RecursiveDepth >NRECURSIONS). I believe you should check your calculation parameters.

**Reply by shihaoye** [reply]
Thanks for your reply,but how should I complete the recursion?（I'm a beginner for Anura 3d)
Thanks for your answer again.

**Comment by EmreDumlu-UM** [comment]
Hi, Anura3D code completes recursion itself it there are not any conflicts in the calculation/material parameters. However, the error you ran into can be gotten through by looking at the parameters one by one. I am not sure which settings you used. Best wishes,Emre From: shihaoyeSent: Wednesday, March 13, 2024 5:29 AMTo: Anura3D/Anura3D_OpenSourceCc: Emre; CommentSubject: Re: [Anura3D/Anura3D_OpenSource] I want to learn about time steps and load steps (Discussion #198) Thanks for your reply,but how should I complete the recursion?（I'm a beginner for Anura 3d)Thanks for your answer again.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
